package.json

```diff
@@ -2,10 +2,10 @@
   "name": "nodejs-katas",
   "version": "1.0.0",
   "description": "Basic Node.js exercises",
-  "main": "index.js",
+  "main": "src/index.js",
   "scripts": {
-    "start": "node index.js",
-    "lint": "eslint test",
+    "start": "node src/index.js",
+    "lint": "eslint src test",
     "test": "mocha test/**/*-test.js",
```

Comment:
Our CI system misses linting functionality. Could you please modify the "test" task to run linting before tests?